03-react/02-users-list/src/main.tsx

```diff
@@ -0,0 +1,283 @@
+		age: 74,
+		city: "New Maryjaneberg",
+		ip: "85.232.129.225",
+		isAdmin: true,
+	},
+	{
+		id: 8,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/woodsman001_128.jpg",
+		first_name: "Cameron",
+		last_name: "Veum",
+		age: 16,
+		city: "Altadena",
+		ip: "168.252.158.225",
+		isAdmin: false,
+	},
+	{
+		id: 9,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/markwienands_128.jpg",
+		first_name: "Tremaine",
+		last_name: "Sanford",
+		age: 10,
+		city: "Lenexa",
+		ip: "38.83.7.48",
+		isAdmin: false,
+	},
+	{
+		id: 10,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/ah_lice_128.jpg",
+		first_name: "Ronaldo",
+		last_name: "Weissnat",
+		age: 83,
+		city: "Baumbachtown",
+		ip: "50.218.254.52",
+		isAdmin: false,
+	},
+	{
+		id: 11,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/zaki3d_128.jpg",
+		first_name: "Ephraim",
+		last_name: "Goyette",
+		age: 63,
+		city: "North Martina",
+		ip: "147.105.193.65",
+		isAdmin: false,
+	},
+	{
+		id: 12,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/boxmodel_128.jpg",
+		first_name: "Clay",
+		last_name: "Kunde",
+		age: 113,
+		city: "South Damon",
+		ip: "209.104.243.157",
+		isAdmin: false,
+	},
+	{
+		id: 13,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/VinThomas_128.jpg",
+		first_name: "Melisa",
+		last_name: "Leannon",
+		age: 18,
+		city: "Hilo",
+		ip: "103.82.167.168",
+		isAdmin: false,
+	},
+	{
+		id: 14,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/miguelmendes_128.jpg",
+		first_name: "Clovis",
+		last_name: "Medhurst",
+		age: 15,
+		city: "Harveybury",
+		ip: "168.223.235.220",
+		isAdmin: false,
+	},
+	{
+		id: 15,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/mandalareopens_128.jpg",
+		first_name: "Mylene",
+		last_name: "Renner",
+		age: 49,
+		city: "Arlington",
+		ip: "223.89.148.36",
+		isAdmin: false,
+	},
+	{
+		id: 16,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/ma_tiax_128.jpg",
+		first_name: "Marcos",
+		last_name: "Ferry",
+		age: 47,
+		city: "Strackehaven",
+		ip: "74.94.165.210",
+		isAdmin: false,
+	},
+	{
+		id: 17,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/balakayuriy_128.jpg",
+		first_name: "Brain",
+		last_name: "Mohr",
+		age: 54,
+		city: "Carrollton",
+		ip: "11.121.113.44",
+		isAdmin: false,
+	},
+	{
+		id: 18,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/aleclarsoniv_128.jpg",
+		first_name: "Bella",
+		last_name: "VonRueden",
+		age: 18,
+		city: "Columbia",
+		ip: "224.144.68.251",
+		isAdmin: true,
+	},
+	{
+		id: 19,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/andrewarrow_128.jpg",
+		first_name: "Franz",
+		last_name: "Raynor",
+		age: 28,
+		city: "Garrickchester",
+		ip: "91.159.111.88",
+		isAdmin: false,
+	},
+	{
+		id: 20,
+		avatar:
+			"https://cdn.fakercloud.com/avatars/carlosgavina_128.jpg",
+		first_name: "Celestino",
+		last_name: "Bailey",
+		age: 61,
+		city: "Aronport",
+		ip: "242.25.16.144",
+		isAdmin: false,
+	},
+];
+
+
+function List(props: UserProps) {
```

Comment:
I would call this component `User` instead of `List` because it shows a single user, not a list.

Reply:
Changed sir, please check.

Reply:
Good!